### PR TITLE
Updated Block Meta

### PR DESCRIPTION
**Pull request by fboucquez**
Fixes https://github.com/nemtech/symbol-openapi/issues/221

### CODE REVIEW

**Comment by Vektrat** [comment]
i think it'd be worth it to describe that totalTransactions counts embedded transactions while the regular transactinsCount does not @segfaultxavi ?

**Review by Vektrat — CHANGES_REQUESTED** [review]
i think it'd be worth it to describe that totalTransactions counts embedded transactions while the regular transactinsCount does not @segfaultxavi ?

**Comment by segfaultxavi** [comment]
Definitely! An example is not enough, `description` is needed everywhere.

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Please use `description` fields.

**Review by rg911 — DISMISSED** [review]

**Comment by fboucquez** [comment]
Can you upgrade the descriptions @segfaultxavi? I can't now

**Comment by segfaultxavi** [comment]
Ok, I'll do it tomorrow. Or is it urgent?

**Review by rg911 — DISMISSED** [review]

**Review by segfaultxavi — APPROVED** [review]
I have no further documentation concerns.

File: spec/core/block/schemas/BlockMetaDTO.yml [hunk 1]
```diff
@@ -16,9 +18,23 @@ properties:
...
       $ref: "../../../schemas/Hash256.yml"
-  numTransactions:
+  totalTransactionsCount:
     type: integer
-    example: 0
-  numStatements:
+    description: >
+      Total number of [transactions](https://docs.symbolplatform.com/concepts/transaction.html)
+      present in this block, including *embedded* transactions
+      (i.e. transactions contained within aggregate transactions).
```
Comment (Vektrat) [inline spec/core/block/schemas/BlockMetaDTO.yml:26]: i still think this is too verbose for here

i'd go with stuff like:

_Total number of transactions that were confirmed in this block, including *embedded* transactions_

links are not necessary in here imo, but your choice

File: spec/core/block/schemas/BlockMetaDTO.yml [hunk 2]
```diff
@@ -16,9 +18,23 @@ properties:
...
+    description: >
+      Number of [transactions](https://docs.symbolplatform.com/concepts/transaction.html)
+      present in this block. This does not count *embedded* transactions
+      (i.e. transactions contained within aggregate transactions).
+    example: 1
+  statementsCount:
     type: integer
+    description: >
+      Number of [transaction statements](https://docs.symbolplatform.com/concepts/receipt.html#transaction-statement)
+      present in this block.
```
Comment (Vektrat) [inline spec/core/block/schemas/BlockMetaDTO.yml:39]: probably needs clarification on what "statements present in the block" means, just like transactions present means "confirmed"

File: spec/core/block/schemas/BlockMetaDTO.yml (hunk 1 above)
Comment (segfaultxavi) [reply spec/core/block/schemas/BlockMetaDTO.yml:26]: Links do not hurt expert readers (just don't click on them), and they definitely help newcomers.

Regarding verbosity, I have the feeling most of the current docs can only be understood by experts and are not very kind to newcomers. If we are to extend our user base we should change that...

In this case it's only a parenthesis with 5 words so I don't think it's too much burden for experts :)

**Review by Vektrat — APPROVED** [review]
approving because there is hurry, we expect pending comments in descriptions to be addressed in another issue

**Pull request merged**